src/index.html

```diff
@@ -1,13 +1,62 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./index.css">
+    <link href="https://fonts.googleapis.com/css2?family=Corinthia:wght@400;700&family=Roboto:wght@400;500;700&display=swap" 
+          rel="stylesheet"
+    >
+  </head>
+  <body>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27'' Retina 5K Monoblock (MNED2UA/A)"
+      >
+      <h1 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+      <p class="card__code">
+        Product code: 195434
+      </p>
+      <div class="card__reviews">
+        <span class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </span>
+        <p class="card__reviews-text">
+          Reviews: 5
+        </p>
+      </div>
+      <div class="card__price">
+        <p class="card__price-text">
+          Price:
+        </p>
+        <p class="card__price-money">
+          $2,199
+        </p>
+      </div>
+      <div 
+        class="card__buy"
+        data-qa="hover"
+      >
+        <a
+          class="card__buy-button"
+          href="#AddToCart"
+        >
+          Buy
+        </a>
+      </div>
```

Comment:
Why do you need this div container for the button? could you do the same without it?